lib/server.js

```diff
@@ -445,10 +443,10 @@ Server.prototype.attach = function(server, options){
         // but by adding a handler, we prevent that
         // and if no eio thing handles the upgrade
         // then the socket needs to die!
-        setTimeout(function() {
-           if (socket.writable && socket.bytesWritten <= 0) {
-             return socket.end();
-           }
+        setTimeout(function () {
+          if (socket.writable && socket.bytesWritten <= 0) {
+            return socket.end();
+          }
         }, options.destroyUpgradeTimeout);
```

Comment:
This line here. If it were to use the `destroyUpgradeTimeout` in line 411, it should default to 1000ms. But because it is using the `destroyUpgradeTimeout` from `options`, if it is undefined, it will default to 0ms.

So my question is, which is the intended behavior, if `options.destroyUpgradeTimeout` is `undefined`? Defaults to 1000 or 0 ms?